sources/PPU/Background.cpp

```diff
@@ -105,17 +108,16 @@ namespace ComSquare::PPU
 			// TODO function to read 2 bytes (LSB order or bits reversed)
 			uint16_t tileMapValue = this->_vram.read(vramAddress);
 			tileMapValue += this->_vram.read(vramAddress + 1) << 8U;
-			this->_drawBgTile(tileMapValue, {
-				(pos.x * this->_characterNbPixels.x) + offset.x,
-				(pos.y * this->_characterNbPixels.y) + offset.y
-			});
+			this->_drawTile(tileMapValue, {(offset.x * NbCharacterWidth) + pos.x,
```

Comment:
Don't you want to multiply the pos by the number of pixels, like before?

Reply:
Nah, I use tile coordinates now instead of pixel offset